### PR TITLE
Refactor description of element container and fix subtle bugs

### PR DESCRIPTION
**Pull request by andreastt**
This change is [Reviewable](https://reviewable.io/reviews/w3c/webdriver/576)

### CODE REVIEW

File: webdriver-spec.html [hunk 1]
```diff
@@ -4376,6 +4379,24 @@
  <li><p>Return false.
 </ol>
 
+<p>An <a>element</a>’s <dfn>container</dfn> is:
+
+<dl class=switch>
+ <dt><a><code>option</code> element</a> in a valid <a>element context</a>
```
Comment (shs96c) [inline webdriver-spec.html]: Also `optgroup`

File: webdriver-spec.html [hunk 2]
```diff
@@ -4882,59 +4903,46 @@
...
+ <li><p><a>Scroll into view</a>
+  the <var><a>element</a></var>’s <a>container</a> <a>element</a>.
 
  <li><p>Wait in an implementation-specific way up to the <a>session
-  implicit wait timeout</a> for <var>container</var> to
+  implicit wait timeout</a> for the <a>container</a> to
   become <a>pointer-interactable</a>.
 
- <li><p>If <var>container</var> is not <a>pointer-interactable</a>,
+ <li><p>If <a>element</a>’s <a>container</a> is not <a>pointer-interactable</a>,
```
Comment (shs96c) [inline webdriver-spec.html]: &lt;var>element&lt;/var>

File: webdriver-spec.html [hunk 3]
```diff
@@ -4882,59 +4903,46 @@
...
-  </dl>
-  </li>
+ <li><p>If the <var>element</var> is
+  an <a><code>input</code> element</a> in the <a>file upload state</a>
+  or it does not have a <a>container</a> <a>element</a>,
+  return <a>error</a> with <a>error code</a> <a>invalid argument</a>.
 
- <li><p><a>Scroll into view</a> the <var>container</var>.
+ <li><p><a>Scroll into view</a>
+  the <var><a>element</a></var>’s <a>container</a> <a>element</a>.
```
Comment (shs96c) [inline webdriver-spec.html]: Also "&lt;var>element&lt;/var>"

Comment (andreastt) [inline webdriver-spec.html]: The last instance here isn’t referring to the variable _element_, but to the concept.

File: webdriver-spec.html (hunk 1 above)
Comment (andreastt) [inline webdriver-spec.html]: Added match.

File: webdriver-spec.html (hunk 3 above)
Comment (andreastt) [inline webdriver-spec.html]: Second instance refers to the concept element, and `<var>` is used for the first.

File: webdriver-spec.html (hunk 2 above)
Comment (andreastt) [inline webdriver-spec.html]: Fixed.

**Pull request merged**